bld/CLMBuildNamelist.pm

```diff
@@ -4564,14 +4564,18 @@ sub setup_logic_exice {
   my $use_exice = $nl->get_value( 'use_excess_ice' );
   my $use_exice_streams = $nl->get_value( 'use_excess_ice_streams' );
   # IF excess ice streams is on
+  if ( (not defined($use_exice_streams) && value_is_true($use_exice)) && ($nl_flags->{'clm_start_type'} == /cold/  || $nl_flags->{'clm_start_type'} == /arb_ic/ ) ) {
```

Comment:
Not quite sure these are sufficient. Might need to use `finidat == ''` in the condition.

Reply:
Logic was updated to use finidat, which is likely more robust.

Reply:
@mvdebolskiy The one thing here is that it looks like potentially the user might set use_excess_ice_streams one way and the logic here might end up overriding what they did. It's better to only set it if the user didn't touch it, and abort with an error if the user tried to do something nonsensical.

Reply:
OK, I adjusted a few things and think it's good now.

Reply:
@mvdebolskiy I adjusted the namelist a bit, feel free to look it over and give feedback. And if you'd like to meet we can do that.

Reply:
Resolving this for now so we'll be able to merge when needed.